universum/configuration_support.py

```diff
@@ -13,6 +16,47 @@
 skip_attributes = {"children", "critical", "skip_numbering_level"}
 
 
+ProjectConfigKeyType = Literal["name", "command", "environment", "artifacts", "report_artifacts",
+                               "artifact_prebuild_clean", "directory", "critical", "background", "finish_background",
+                               "code_report", "pass_tag", "fail_tag", "if_env_set"]
+
+
+class ProjectConfig(TypedDict, total=True):
+    """
+    :param name: Specifies the name of the configuration.
+    :type name: str
+    The name is used as the title of the build log block corresponding to the build of this configuration. It is
+    also used to generate name of the log file if the option for storing to log files is selected. A project
+    configuration can have no name, but it is not recommended for aesthetic reasons. If several project
+    configurations have the same names, and logs are stored to files (see --launcher-output / -lo command-line
+    parameter for details), all logs for such configurations will be stored to one file in order of their appearances.
+
+    :param command: The list with command line for the configuration launch.
+    :type command: list
+    For every project configuration first list item should be a console command (e.g. script name, or any other
+    command like ls), and other list items should be the arguments, added to the command (e.g. --debug or -la). Every
+    command line element, separated with space character, should be passed as a separate string argument. Lists like
+    ["ls -a"] will not be processed correctly and thus should be splat into ["ls", "-a"]. Lists like ["build.sh",
+    "--platform A"] will not be processed correctly and thus should be plat into ["build.sh", "--platform",
+    "A"]. A project configuration can have an empty list as a command. Such configuration won’t do anything except
+    informing user about missing it.
```

Comment:
Looks like you took the description from documentation; but it doesn't look nice now that I'm reading it. Can you, maybe, rephrase it a little to make it more understandable? Especially the thing about spaces and _arguments_
(You may even apply the same changes to actual documentation paragraph you took this description from in the same PR)

Reply:
ok, I'll include these changes in the PR

Reply:
it was decided to move documentation changes in separate PR